.github/workflows/error-prone.yml

```diff
@@ -0,0 +1,32 @@
+name: Error-Prone
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  error-prone:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Install groovy
+        run: sudo apt install groovy
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Execute Error-Prone on compile phase
+        run: |
+          groovy ./.ci/error-prone-check.groovy compile
```

Comment:
Why can't this be single line? Same for others below.

Reply:
Done